### PR TITLE
MNT: Re-rendered with conda-smithy 1.4.6

### PR DESCRIPTION
**Pull request by lesteve**
Close #6.

### CODE REVIEW

**Comment by lesteve** [comment]
@ocefpaf it seems like everything is fine with the latest version of conda-smithy so I'll merge this one.

**Comment by ocefpaf** [comment]
> @ocefpaf it seems like everything is fine with the latest version of conda-smithy so I'll merge this one.

:+1:

File: appveyor.yml [hunk 1]
```diff
@@ -20,22 +20,8 @@ environment:
...
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 35
```
Comment (jakirkham) [inline appveyor.yml:38]: Should the Python matrix be dropped? 😕

If not, could you please try manually fixing this? Seems all CI files have this issue.

Comment (jakirkham) [reply appveyor.yml:38]: Should be fixed in PR ( https://github.com/conda-forge/nilearn-feedstock/pull/9 ).

**Pull request merged**